mkdocs.yml

```diff
@@ -105,6 +105,7 @@ nav:
   - Blog:
       - 2020:
           - '7.0 Release': blog/2020/7.0-release.md
+          - 'A guide on disabling/enabling lsp-mode features': blog/2020/how-to-turn-off.md
```

Comment:
This probably it's a tutorial now right? not a blog post

Reply:
IDK - but we have to link it from the readme, it may go in the main tree as well.

Reply:
We can link it from the FAQ section, but I didn't get it why this should not be in the tutorials section?